docs/admin/runai-setup/self-hosted/ocp/backend.md

```diff
@@ -31,34 +31,60 @@ Run the helm command below:
 | `thanos.receive.persistence.storageClass` and `postgresql.primary.persistence.storageClass` | Storage class | The installation to work with a specific storage class rather than the default one |
-| `global.imagePullSecrets:` <br> &ensp; `- name: <secret-name>`  | Docker secret | Provide credentials for accessing the organization's docker registry. This is required for air-gapped environments  |
-| `<component>` <br> &ensp;`resources:` <br> &emsp; `limits:` <br> &emsp; &ensp; `cpu: 500m` <br> &emsp; &ensp; `memory: 512Mi` <br> &emsp; `requests:` <br> &emsp; &ensp; `cpu: 250m` <br> &emsp; &ensp; `memory: 256Mi`  | Pod request and limits  |  `<component>` may be anyone of the following: `backend`, `frontend`, `assetsService`, `identityManager`, `tenantsManager`, `keycloakx`, `grafana`, `authorization`, `orgUnitService`,`policyService`  |   
-|<div style="width:200px"></div>| | |Use the `--set` syntax in the helm command above.  
+| `<component>` <br> &ensp;`resources:` <br> &emsp; `limits:` <br> &emsp; &ensp; `cpu: 500m` <br> &emsp; &ensp; `memory: 512Mi` <br> &emsp; `requests:` <br> &emsp; &ensp; `cpu: 250m` <br> &emsp; &ensp; `memory: 256Mi`  | Pod request and limits  |  `<component>` may be anyone of the following: `backend`, `frontend`, `assetsService`, `identityManager`, `tenantsManager`, `keycloakx`, `grafana`, `authorization`, `orgUnitService`,`policyService`  |
+|<div style="width:200px"></div>| | |
+
+Use the `--set` syntax in the helm command above.
+
+!!! Note
+    If you modify one of the usernames or passwords (KeyCloak, PostgreSQL, Grafana) after Run:ai is already installed, perform the following steps to apply the change:
+
+    1. Modify the username/password within the relevant component as well (KeyCloak, PostgreSQL, Grafana).
+    2. Run `helm upgrade` for Run:ai with the right values, and restart the relevant Run:ai pods so they can fetch the new username/password.
+
+## Next steps
+
+### Connect to Run:ai user interface
+
+* Run: `oc get routes -n runai-backend` to find the Run:ai Administration User Interface URL.
+* Log in using the default credentials: User: `test@run.ai`, Password: `Abcd!234`.
+* Go to the Users area and change the password.
+
+### Enable Forgot Password (optional)
 
-## Connect to Run:ai User Interface
+To support the *Forgot password* functionality, follow the steps below.
 
-* Run: `oc get routes -n runai-backend` to find the Run:ai Administration User Interface URL. 
-* Log in using the default credentials: User: `test@run.ai`, Password: `Abcd!234`. 
-* Go to the Users area and change the password. 
+* Go to `runai.<openshift-cluster-domain>/auth` and Log in.
+* Under `Realm settings`, select the `Login` tab and enable the `Forgot password` feature.
+* Under the `Email` tab, define an SMTP server, as explained [here](https://www.keycloak.org/docs/latest/server_admin/#_email){target=_blank}
 
-## Next Steps
+### Install Run:ai Cluster
```

Comment:
```suggestion
## Next steps
```

@jasonnovichRunAI it wasn't part of my PR so I changed it back to how it was... if you are the one changing it on purpose, ignore this comment